src/main/interface/cli.c

```diff
@@ -2580,6 +2580,73 @@ static void cliBeeper(char *cmdline)
 }
 #endif
 
+#ifdef USE_TELEMETRY
+static void printTelemetry(uint8_t dumpMask, const telemetryConfig_t *telemetryConfig, const telemetryConfig_t *defaultTelemetryConfig)
```

Comment:
All of this code can be replaced with CLI variable (`uint32_t enabledSensors`) and some bit fiddling on user side. It won't be that user friendly, but it will save considerable amount of FLASH.

And eventually CLI `set` may be improved to handle this seamlessly.

Reply:
Do you mean I should call printTelemetry with telemetryConfig->sensorsEnabled rather than passing telemetryConfig itself?

Reply:
Just expose `enabledSensors` as CLI `set` variable

Reply:
Oh, I see, like SET TELEMETRY_SENSORS=7.